Client/mods/deathmatch/logic/CClientVehicle.cpp

```diff
@@ -1464,6 +1464,30 @@ void CClientVehicle::SetWheelStatus(unsigned char ucWheel, unsigned char ucStatu
 
                 // Update the wheel's visibility
                 m_pVehicle->SetWheelVisibility((eWheelPosition)ucWheel, (ucStatus != DT_WHEEL_MISSING));
+
+                // restart wheels component visibility (fix #1556)
+                std::map<SString, SVehicleComponentData>::iterator iter = m_ComponentData.begin();
+
+                for (; iter != m_ComponentData.end(); iter++)
+                {
+                    SString strComponentName = (*iter).first;
+                    if (strComponentName.BeginsWith("wheel_"))
```

Comment:
You're creating a new SString object for `"wheel_"` in every iteration. To eliminate this you can create a static string object somewhere and use it here.